### PR TITLE
Expiry Field Validation Issue

### PR DESCRIPTION
**Pull request by Arslan007**
There was one glitch in current code. When you enter all details properly like Name, Credit Card Number, Expiry and CVV then control gives your proper validation result But if you give all the values properly and edit the Expiry field 11 / 20 and keep pressing digits although nothing displays on UI but it makes input field invalid

### CODE REVIEW

**Comment by Arslan007** [comment]
Hi Adyen Team,

Can you please review my Pull Request

File: Checkout/CheckoutCardUI.swift [hunk 1]
```diff
@@ -255,7 +255,13 @@ open class CardPaymentField: CheckoutPaymentFieldView, UITextFieldDelegate {
             
             //            delegate?.fieldChangedValue(self.numberField, valid: valid)
             delegate?.paymentFieldChangedValidity(valid)
-        } else if textField == self.expirationField {
+        } else if textField == self.expirationField
+        {
+            // When Expiry is added properly like this 11/20 and if you press any number then it makes expiry field invalid even nothing displayed on UI
+            if(string.characters.count != 0 && textField.text?.characters.count == 7)
```
Comment (olutsenko) [inline Checkout/CheckoutCardUI.swift]: Could you please explain why do you use value '7' in this case?

Comment (Arslan007) [inline Checkout/CheckoutCardUI.swift]: I am using value 7 specifically due to Expiry data format. i.e. "11 / 20" in expiry field.
There are two spaces plus forward slash "/" so it makes value 7 along with expiry.

**Comment by olutsenko** [comment]
Merged, @Arslan007 thanks once again!

File: Checkout/CheckoutCardUI.swift (hunk 1 above)
Comment (Arslan007) [inline Checkout/CheckoutCardUI.swift]: Thanks!!! Much Appreciated

**Pull request merged**